proto/metadata.proto

```diff
@@ -273,8 +273,6 @@ message Metadata {
     bool bkup_arbitration_resp_code = 82;
     // Device requires IPOverIP decapsulation for backup NHG without interfaces.
     bool backup_nhg_requires_vrf_with_decap = 83;
-    // Devices don't support ISIS Timers lsp-refresh-interval leaf.
```

Comment:
Please follow all the steps in `Removing Deviations` section here: https://github.com/openconfig/featureprofiles/tree/main/internal/deviations#removing-deviations

For this PR, I still need to do the following:
 - add `reserved 84;` at the end of this Deviations message
 - Run the `make proto..` for updating metadata.pb.go

Reply:
@prinikasn , I did add reserved 84  and tried to run  "make proto/metadata_go_proto/metadata.pb.go" . It gives error as metadata.pb.go is not executable file.  Should i manually change permissions on file and re run this ?

Also there is already reserved field available, updated it as below. please let me know if this is fine.


  message Platform {
    // Vendor of the device.
    ondatra.Device.Vendor vendor = 1;
    // Regex for hardware model of the device.
    // The empty string will match any hardware model.
    string hardware_model_regex = 3;
    // Regex for software version of the device.
    // The empty string will match any software version.
    string software_version_regex = 4;
    // Reserved field numbers and identifiers.
    reserved 2, 84; ============================> Added 84 here.
    reserved "hardware_model";
  }


Error:
==========
mkdir -p proto/metadata_go_proto
# Set directory to hold symlink
mkdir -p protobuf-import
# Remove any existing symlinks & empty directories
find protobuf-import -type l -delete
find protobuf-import -type d -empty -delete
# Download the required dependencies
go mod download
# Get ondatra modules we use and create required directory structure
go list -f 'protobuf-import/{{ .Path }}' -m github.com/openconfig/ondatra | xargs -L1 dirname | sort | uniq | xargs mkdir -p
go list -f '{{ .Dir }} protobuf-import/{{ .Path }}' -m github.com/openconfig/ondatra | xargs -L1 -- ln -s
protoc -I='protobuf-import' --proto_path=proto --go_out=./ --go_opt=Mmetadata.proto=proto/metadata_go_proto metadata.proto
protoc-gen-go: program not found or is not executable
--go_out: protoc-gen-go: Plugin failed with status code 1.
make: *** [Makefile:42: proto/metadata_go_proto/metadata.pb.go] Error 1

Reply:
I have installed protoc-gen-go:, re ran make and updated . Please check if it is good now.